cluster/gce/gci/configure-helper.sh

```diff
@@ -1268,6 +1268,15 @@ current-context: service-account-context
 EOF
 }
 
+function create-kube-scheduler-config {
+  echo "Creating kube-scheduler config file"
+  mkdir -p /etc/srv/kubernetes/kube-scheduler
+  cat <<EOF >/etc/srv/kubernetes/kube-scheduler/config
```

Comment:
would this be better suited for `/etc/kubernetes`?

I generally don't think of `srv` as configuration but rather content served on user request (http server for eg).

Reply:
This is the same location that we use for all kubeconfigs. We mount them to the pod here

https://github.com/kubernetes/kubernetes/blob/9b53b9306e50ffdb1b1b5c930fb4b4ce1f404105/cluster/gce/manifests/kube-scheduler.manifest#L71

Reply:
Gotcha

Reply:
Should this be integrated with create-kubescheduler-policy-config https://github.com/kubernetes/kubernetes/blob/master/cluster/gce/gci/configure-helper.sh#L1271? If not I would like to at least make kube-scheduler vs kubescheduler consistent.